helm/values-dev.yaml

```diff
@@ -40,6 +43,8 @@ microservice-chart:
       QUEUE_CONVERSION_CONNECTION_STRING: "fdr-sa-connection-string"
       EVENT_HUB_RE_CONNECTION_STRING: "azure-event-hub-re-connection-string"
       BLOB_RE_CONNECTION_STRING: "fdr-re-sa-connection-string"
+      BLOB_HISTORY_CONNECTION_STRING: "fdr-history-sa-connection-string"
```

Comment:
use the same key, like `FDR_HISTORY_CONNECTION_STRING` instead of `BLOB_HISTORY_CONNECTION_STRING` and `TABLE_HISTORY_CONNECTION_STRING`